analyzer/secret/testdata/image-config.yaml

```diff
@@ -0,0 +1,9 @@
+rules:
```

Comment:
I thought we would add `path` here so that we can make sure our allowrule works with image scanning.

Reply:
Done